### PR TITLE
NXP-26494: fix duplicate headers in csv export

### PR DESCRIPTION
**Pull request by pierre-gautier**
T&P is running https://qa.nuxeo.org/jenkins/job/TestAndPush/job/ondemand-testandpush-pierregautier/226/
Last failed and is fixed now

### CODE REVIEW

**Comment by kevinleturc** [comment]
👍

File: nuxeo-features/nuxeo-platform-csv-export/src/test/java/org/nuxeo/ecm/platform/csv/export/TestCSVExportAction.java [hunk 1]
```diff
@@ -338,8 +331,15 @@ public void write(int b) throws IOException {
         }
 
         List<String> lines = Files.lines(testCsv.toPath()).collect(Collectors.toList());
+        int headerCount = 0;
+        for (String line : lines) {
+            if (line.startsWith("repository,uid")) {
+                headerCount++;
+            }
+        }
```
Comment (funshodavid) [inline nuxeo-features/nuxeo-platform-csv-export/src/test/java/org/nuxeo/ecm/platform/csv/export/TestCSVExportAction.java]: Could be replaced with : 
```suggestion
        long count = lines.stream().filter(l -> l.startsWith("repository,uid")).count();
```

**Comment by funshodavid** [comment]
:+1:

**Comment by pierre-gautier** [comment]
👍

**Pull request merged**